eth-wallpaper-gen/main.py

```diff
@@ -0,0 +1,70 @@
+import sys
```

Comment:
Can you modify this to be an executable python script?

Reply:
Done :+1: